BioscoopSysteemAPI/BioscoopSysteemAPI/DTOs/MovieDTOs/MovieCreateDTO.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace BioscoopSysteemAPI.DTOs.MovieDTOs
+{
+    public class MovieCreateDTO
```

Comment:
Alle MovieDtos zijn hetzelfde. Zouden we niet gewoon 1 MovieDto kunnen gebruiken? het lijkt erop dat we meerdere movieDtos hebben die allemaal dezelfde implementatie hebben? Of was dit meer bedoeld voor readability?